source/conf.py

```diff
@@ -344,3 +344,37 @@
 # If true, figures, tables and code-blocks are automatically numbered if they
 # have a caption. At same time, the numref role is enabled.
 numfig = True
+
+# -- Custom configuration -------------------------------------------------
+
+from jinja2 import Environment, StrictUndefined
+
+# Use Jinja to parse RST files first
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    jinja_env = Environment(undefined=StrictUndefined)
```

Comment:
I still don't think we should modify the template delimiters as it would make working with it more confusing in the future. As we'd have to translate all documentation and tutorials on Jinja2 into the new formatting within our heads.